.nvmrc

```diff
@@ -1 +1 @@
-18.20.4
+18.20.5
```

Comment:
Can we change the node version to 18.20.6 here and everywhere else in this PR? That is what main is on now and I can't test this locally without downgrading my docker, etc

Reply:
done.

Reply:
ok, pulling down now to run locally